docs/adding-suggestions.md

```diff
@@ -176,8 +176,16 @@ public static List<SuggestionProvider> getAllSuggestionProviders(boolean verbose
 }
 ```
 
+## Result
+
 We now have a working [Suggestion Provider](./library-architecture.md#suggestion-provider) utilizing the data from our [Data Provider](./library-architecture.md#data-provider)!
 
+Running the profile analyzer on a Bazel profile with major garbage collection events will produce output similar to:
+
+![Output of the new Suggestion Provider](https://github.com/EngFlow/bazel_invocation_analyzer/blob/main/docs/assets/gc-suggestion-output.png?raw=true)
```

Comment:
What is the advantage to adding an image, rather than including a code block here?

Reply:
The highlighting of titles and coloring of text. I couldn't find a way to do that in-line with a mono-spaced font in markdown.  It all blends together in a mush without the styling. If you know of a way, I'd like to hear it!

Reply:
Yes, that makes sense. Unfortunately, I do not know of a way.